subprojects/language-java/src/integTest/groovy/org/gradle/api/tasks/compile/JavaCompileJavaVersionIntegrationTest.groovy

```diff
@@ -37,15 +37,18 @@ class JavaCompileJavaVersionIntegrationTest extends AbstractIntegrationSpec {
      */
     def setup() {
         executer.requireDaemon().requireIsolatedDaemons()
+        executer.beforeExecute {
+            noDeprecationChecks()
```

Comment:
❓ I suppose that is because Java 1.7 is deprecated for compilation with Java 17? Is targeting Java 1.7 important for the test? Could we target Java 8 instead?

Reply:
We will shortly run into a similar issue with Java 21, which deprecates 8.